### PR TITLE
py-pydicom: New package.

### PR DESCRIPTION
**Pull request by eborisch**
Not sure if you like new packages submitted via this mechanism, but thought I would give it a shot... It's my first attempt, so please let me know conventions / best practices I've missed.

### CODE REVIEW

File: graphics/py-pydicom/Makefile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# $NetBSD$
+
+DISTNAME=	pydicom-1.2.0
+PKGNAME=	${PYPKGPREFIX}-${DISTNAME}
+CATEGORIES=	graphics python
+MASTER_SITES=	${MASTER_SITE_GITHUB:=pydicom/}
+GITHUB_PROJECT=	pydicom
```
Comment (krytarowski) [inline graphics/py-pydicom/Makefile]: Does it still work if you drop this line?

File: graphics/py-pydicom/Makefile [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+GITHUB_TAG=	v${PKGVERSION_NOREV}
+
+MAINTAINER=	eborisch@gmail.com
+HOMEPAGE=	https://github.com/pydicom/pydicom/
+COMMENT=	Pure python package for working with DICOM files
+LICENSE=	modified-bsd AND mit
+
+WRKSRC=	${WRKDIR}/pydicom-1.2.0
```
Comment (krytarowski) [inline graphics/py-pydicom/Makefile]: Does it still work if you drop this line?

**Comment by iamleot** [comment]
Hello Kamil,

Kamil Rytarowski writes:
> > @@ -0,0 +1,27 @@
> +# $NetBSD$
> +
> +DISTNAME=	pydicom-1.2.0
> +PKGNAME=	${PYPKGPREFIX}-${DISTNAME}
> +CATEGORIES=	graphics python
> +MASTER_SITES=	${MASTER_SITE_GITHUB:=pydicom/}
> +GITHUB_PROJECT=	pydicom
>
> Does it still work if you drop this line?
>

I think it will not work because GITHUB_PROJECT by default is
PKGBASE.

However, I think that for MASTER_SITES, ${MASTER_SITE_PYPI:=p/pydicom/}
can be used instead and will avoid to define GITHUB_*.

**Comment by krytarowski** [comment]
GitHub as upstream is good because we can detect easily newer versions with pkglint.

**Comment by eborisch** [comment]
> Hello Kamil, Kamil Rytarowski writes:
> I think it will not work because GITHUB_PROJECT by default is PKGBASE. However, I think that for MASTER_SITES, ${MASTER_SITE_PYPI:=p/pydicom/} can be used instead and will avoid to define GITHUB_*.

I started with that approach, but the fetch fails. Not being all that familiar with pkgsrc's internals, I had difficulty tracking down the source of the error, so I aborted and went with the Github download.

File: graphics/py-pydicom/Makefile (hunk 2 above)
Comment (eborisch) [reply graphics/py-pydicom/Makefile]: Yes. Done.

**Comment by bsiegert** [comment]
If the upstream name is pydicom, the package name should be py-dicom, not
py-pydicom.

Am Mi., 31. Okt. 2018, 15:16 hat Eric A. Borisch <notifications@github.com>
geschrieben:

**Comment by iamleot** [comment]
Benny Siegert writes:
> If the upstream name is pydicom, the package name should be py-dicom, not
> py-pydicom.

On pypi it seems that there is also a (probably different and/or older
version) py-dicom package so IMHO it would be better to be a bit
cacophonic and keep the name as py-pydicom (I think that an older
version is present in pkgsrc-wip as wip/py-dicom).

Eric, a possible further (but tiny!) changes, but it would be
nice to discuss that before importing:

 - Would it be better/easier/more consesistent with other packages to
   change the category to `biology' instead of `graphics'? (wip/py-dicom
   package has `biology' as category)

Possible little nitpicks:

 - It would be better to use `+=' to append to PLIST_SUBST instead of
   `=' (I think that actually the pkgsrc infrastructure - at least in
   that case - handles that in the same way, but I think that
   reinitializing it via `=' can be problematic in some cases)
 - (This can be seen when it is installed with PKG_DEVELOPER=yes)
   Several *.dcm files are installed with the execute permission set,
   as a short term workaround it would be nice to adjust the permissions.
   A longer term solution would be to contact upstream about that, I
   think that these files aren't executed and so should not need execute
   bit set.

After discussing/addressing these I think that py-pydicom package can be
imported!


Thank you!

**Comment by eborisch** [comment]
> On pypi it seems that there is also a (probably different and/or older
version) py-dicom package so IMHO it would be better to be a bit
cacophonic and keep the name as py-pydicom (I think that an older
version is present in pkgsrc-wip as wip/py-dicom).

Yes, they are two [distinct packages from the same author](https://pydicom.github.io/pydicom/stable/transition_to_pydicom1.html) designed to enable side-by-side access to the old API during transitions.

**Comment by eborisch** [comment]
OK. Moved to biology. Fixed stray +x perms. Moved to += for PLIST_SUBST.

Thanks for the reviews. Anything else I missed?

**Comment by iamleot** [comment]
"Eric A. Borisch" writes:
> OK. Moved to biology. Fixed stray +x perms. Moved to += for PLIST_SUBST.
>
> Thanks for the reviews. Anything else I missed?

Nope, that looks good!

I have only changed `chmod' with `${CHMOD}' and imported it in
pkgsrc as biology/py-pydicom.


Thank you very much Eric for packaging it!

**Pull request closed without merging**